src/batch-client/index.ts

```diff
@@ -37,6 +37,7 @@ import {
 	FolderView,
 	InviteReplyInput,
 	PreferencesInput,
+	PropertiesInput,
```

Comment:
remove this as we are using metadata to save smime operation

Reply:
Done